src/coreclr/interpreter/compiler.h

```diff
@@ -305,6 +305,7 @@ struct InterpBasicBlock
     int32_t ilOffset, nativeOffset;
     int32_t nativeEndOffset;
     int32_t stackHeight;
+    uint32_t bbEHClauseIndex;
```

Comment:
I wonder if we really actually need the bbEHClauseIndex. If I understand it correctly, it serves for distinguishing between BBs of the leave chain islands from the regular blocks that share the same IL offset. But the BBs of the leave chain islands would never be merged with any other BBs, because they only generate optional INTOP_CALL_FINALLY and INTOP_BR or INTOP_LEAVE_CATCH and can be only entered by a INTOP_BR.